src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -22,6 +22,51 @@
 
 #include <stdio.h>
 
+/**
+ * @def CHIP_CONFIG_ICD_CLIENT_INFO_KEY_CONST
+ *
+ * @brief Defines ICD Client Info Key const
+ */
+#ifndef CHIP_CONFIG_ICD_CLIENT_INFO_KEY_CONST
+#define CHIP_CONFIG_ICD_CLIENT_INFO_KEY_CONST "g/%x/icdkey"
+#endif
+
+/**
+ * @def CHIP_CONFIG_ICD_CLIENT_INFO_COUNT
+ *
+ * @brief Defines ICD Client Info count formater
+ */
+#ifndef CHIP_CONFIG_ICD_CLIENT_INFO_COUNT
+#define CHIP_CONFIG_ICD_CLIENT_INFO_COUNT "g/%x/icdcount"
+#endif
+
+/**
+ * @def CHIP_CONFIG_ICD_CLIENT_INFO_MAX_SIZE
+ *
+ * @brief Defines ICD Client Info count formater
+ */
+#ifndef CHIP_CONFIG_ICD_CLIENT_INFO_MAX_SIZE_CONST
+#define CHIP_CONFIG_ICD_CLIENT_INFO_MAX_SIZE_CONST "g/icdsize"
+#endif
+
+/**
+ * @def CHIP_CONFIG_ICD_FABRIC_LIST
+ *
+ * @brief Defines ICD Fabric List
+ */
+#ifndef CHIP_CONFIG_ICD_FABRIC_LIST
+#define CHIP_CONFIG_ICD_FABRIC_LIST "g/icdfl"
+#endif
+
+/**
+ * @def CHIP_CONFIG_ICD_FABRIC_LIST_COUNTER
+ *
+ * @brief Defines ICD Fabric List Counter
+ */
+#ifndef CHIP_CONFIG_ICD_FABRIC_LIST_COUNTER
+#define CHIP_CONFIG_ICD_FABRIC_LIST_COUNTER "g/icdflc"
+#endif
```

Comment:
Please remove configurability and make the lists appear *inline* in the methods (like all the other existing cases).

Reply:
updated, thanks